faucet/valve.py

```diff
@@ -833,7 +833,7 @@ def lacp_update(self, port, lacp_up, now=None, lacp_pkt=None, other_valves=None,
         select_updated = self.switch_manager.lacp_update_port_selection_state(
             port, self, other_valves, cold_start=False)
         if updated or select_updated:
-            if updated:
+            if updated or select_updated:
```

Comment:
looks like this 2nd if clause is now redundant and can be removed? line 835 does the same thing.

Reply:
Yup. Got overlooked. Thanks.